depthai_helpers/calibration_utils.py

```diff
@@ -1,868 +0,0 @@
-import cv2
```

Comment:
We shouldn't remove this file, as it doesn't contain DepthAI specific stuff. It is imported by `calibrate.py` - and I think removing that isn't good either, as users would wonder where it disappeared. Instead I think we should print a warning message (and then exit) inside `calibrate.py` that is wasn't migrated to Gen2, and if users want to run the calibration, they should switch to the `gen1_main` branch.

Reply:
@alex-luxonis it contained 
```
import consts.resource_paths
```
which we don't have anymore. But I agree, printing a message or throwing an error instead of removing the file will be better for UX